src/main/java/me/zeroeightsix/kami/module/modules/combat/AutoEZ.kt

```diff
@@ -63,13 +78,12 @@ object AutoEZ : Module() {
             "I like to eat pasta, do you prefer nachos?",
             "Sometimes I sing soppy, love songs in the car.",
             "I love the way your hair glistens in the light",
-            "In my free time I like to watch cat videos on youtube",
             "When I saw the guy with a potion I knew there was trouble brewing.",
             "I enjoy long walks on the beach and playing Hypixel",
-            "Doin a bamboozle fren.",
             "I need help, teach me how to play!",
+            "What happens if I add chocolate milk to macaroni and cheese?",
             "Can you paint with all the colors of the wind"
-    ) // Got these from the forums, kinda based -humboldt123
+    ) // Got these from the forums, kinda based -humboldt123 
 
```

Comment:
You can add your thing back but don't have an extra *indent*

Reply:
Not sure what's the problem now after i just removed my line?

Reply:
no problem